source/documentation/ways-of-working/index.html.md.erb

```diff
@@ -6,3 +6,206 @@ review_in: 3 months
 ---
 
 # <%= current_page.data.title %>
+
+## What are our Core Values?
+
+- **Collaboration:** Foster a culture of open communication and collective problem-solving.
+- **Quality:** Uphold high standards in our work, ensuring reliability and efficiency.
+- **Innovation:** Embrace new ideas and technologies that help us improve our service offering.
+- **Transparency:** Maintain clear visibility into our processes, decisions, and progress.
+
+## Team Roles
+
+- **Team Members:** Comprising a mix of developers, data and platform engineers,
+responsible for the delivery of tasks and active participation in all Agile ceremonies.
+- **Product Manager (PM):** Oversees the product backlog,
+ensuring it aligns with stakeholder needs and business objectives.
+- **Delivery Manager (DM):** Facilitates the Agile process,
+removing blockers and ensuring timelines are met.
+
+## Communication and Delivery
+
+- **Daily [Stand-ups](#stand-up):** A quick get-together to share progress updates.
+- **Ceremonies:** How we make sure our team stays aligned as we work to deliver during Agile Sprints.
+
+## Ceremonies
+
+### Schedule
+
+This team uses **Fortnightly Sprints** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence   | Day             | Time          |
```

Comment:
I would leave out the day and time here - it's gonna change often, and the best place to check this is the team calendar

Reply:
See [other comment](https://github.com/ministryofjustice/data-catalogue-runbooks/pull/11#discussion_r1542639214) - I think it's useful to at least have the order and details here while we're refining our approach, and we can remove the dates/times in the future (maybe at the next review).

Reply:
Sounds like a good plan 👍🏻